### PR TITLE
Added new field for faceEmbeddings

### PR DESCRIPTION
**Pull request by Thejas775**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.  

# Type of change  

- [x] New feature  - Implemented new field for accepting faceEmbeddings for HWC-Mobile-App

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `faceEmbedding` field in beneficiary registration to support additional biometric data.
	- Enhanced the `Beneficiary` class to accommodate the new `faceEmbedding` data structure.
  
- **Bug Fixes**
	- Minor formatting changes made to constructor signatures for improved readability.

These updates improve the system's capability to handle biometric information related to beneficiaries, enhancing the overall functionality.

### CODE REVIEW

**Comment by drtechie** [comment]
@Thejas775 does this one-line change take care of accepting faceEmbedding and sending it back when down-syncing?

Adding @devikasuresh20 and @ravishanigarapu as reviewers.

File: src/main/java/com/iemr/common/data/beneficiary/Beneficiary.java [hunk 1]
```diff
@@ -178,7 +179,7 @@ public class Beneficiary
 
 	/**
 	 * @param beneficiaryRegID
-	 *            the beneficiaryRegID to set
+	 *                         the beneficiaryRegID to set
```
Comment (drtechie) [inline src/main/java/com/iemr/common/data/beneficiary/Beneficiary.java:182]: indent of these comments look off

**Comment by Thejas775** [comment]
Resolved comment indentation issues

File: src/main/java/com/iemr/common/data/beneficiary/Beneficiary.java (hunk 1 above)
Comment (Thejas775) [reply src/main/java/com/iemr/common/data/beneficiary/Beneficiary.java]: I was using an extension of VS code called Prettier due to which it happened. Now, I have changed it.

**Comment by drtechie** [comment]
@coderabbitai review

**Review by devikasuresh20 — COMMENTED** [review]
Consider using camelCase for variable names as it is standard practice in Java.

File: src/main/java/com/iemr/common/controller/beneficiary/BeneficiaryRegistrationController.java [hunk 2]
```diff
@@ -200,7 +200,8 @@ public String createBeneficiary(
 					+ "\"healthCareWorkerID\":\"Short\",\"communityID\":\"Integer\",\"districtID\":\"Integer\",\"stateID\":\"Integer\","
 					+ "\"pinCode\":\"String\",\"blockID\":\"Integer\",\"districtBranchID\":\"Integer\",\"createdBy\":\"String\","
 					+ "\"addressLine1\":\"String\"},\"benPhoneMaps\":{\"parentBenRegID\":\"Long\",\"phoneNo\":\"String\","
-					+ "\"phoneTypeID\":\"Integer\",\"benRelationshipID\":\"Integer\",\"deleted\":\"Boolean\",\"createdBy\":\"String\"}}") @RequestBody BeneficiaryModel beneficiaryModel,
+					+ "\"phoneTypeID\":\"Integer\",\"benRelationshipID\":\"Integer\",\"deleted\":\"Boolean\",\"createdBy\":\"String\"},"
+					+ "\"faceEmbedding\":[\"Float\"]}") @RequestBody BeneficiaryModel beneficiaryModel,
```
Comment (ravishanigarapu) [inline src/main/java/com/iemr/common/controller/beneficiary/BeneficiaryRegistrationController.java:204]: @Thejas775  Beneficiary Model changes needed for faceEmbedding

**Pull request merged**